Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "fantoccini"
-version = "0.18.0-beta.1"
+version = "0.18.0"
```

```suggestion
version = "0.18.0-beta.1"
```
I like to make the actual version bump in its own commit. Coming shortly!